packages/slice-machine/src/theme.ts

```diff
@@ -262,7 +262,7 @@ const AppTheme = (): Theme =>
         fontWeight: "body",
         fontSize: "1",
         color: "textClear",
-        border: (t) => `1px solid ${t?.colors?.borders}`,
+        border: (t) => `1px solid ${String(t?.colors?.borders)}`,
```

Comment:
theme ui is weird with typescript

Reply:
haha, it's what they recommand so that the value that can be undefined is a string